src/Badge/Badge.tsx

```diff
@@ -0,0 +1,68 @@
+import * as classnames from "classnames";
+import * as PropTypes from "prop-types";
+import * as React from "react";
+import _ = require("lodash");
+
+import { Values } from "src/utils/types";
+
+import "./Badge.less";
+
+export interface Props {
+  children: React.ReactNode;
+  className?: string;
+  superscript?: boolean;
+  color?: Values<typeof Color>;
+  size?: Values<typeof Size>;
+}
+
+export const cssClass = {
+  CONTAINER: "Badge",
+  SUPER: "Badge--super",
+  color: c => `Badge--${c}`,
+  size: s => `Badge--${s}`,
+};
+
+const Size = {
+  S: "s",
+  M: "m",
+  L: "l",
+} as const;
+
+const Color = {
+  RED: "red",
+  BLUE: "blue",
+  GRAY: "gray",
+};
+
+const propTypes = {
+  children: PropTypes.node.isRequired,
+  className: PropTypes.string,
+  color: PropTypes.oneOf(_.values(Color)),
+  size: PropTypes.oneOf(_.values(Size)),
+};
```

Comment:
I think that we can just get rid of these React `propTypes` now that we have real TypeScript types in `clever-components`. These used to be more important when we didn't have TypeScript types.